td-shim/Cargo.toml

```diff
@@ -49,9 +48,9 @@ cet-ss = []
 secure-boot = ["der"]
 tdx = ["tdx-tdcall", "td-exception/tdx", "td-logger/tdx", "x86"]
 main = [
-    "elf-loader",
+    "td-loader",
```

Comment:
only one copy of "td-loader" is needed, please remove the duplicated one.

Reply:
Thanks for your comment. I have update the PR.